src/components/nav-bar/nav-bar.styles.scss

```diff
@@ -0,0 +1,26 @@
+.nav-bar-container {
+    width: auto;
+    height: auto;
```

Comment:
just make sure these properties are necessary. if something changes, plop them back in.

Reply:
after testing, it was irrelevant, so its been deleted